distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/ServiceManagerProxy.java

```diff
@@ -38,8 +42,15 @@ public ServiceManagerProxy(ServiceManager serviceManager) {
 
     @Override
     public Object invoke(Object proxy, Method method, Object[] args) throws Throwable {
-
-        Subject subject = SECURITY.runAsAdmin(SECURITY::getSystemSubject);
-        return subject.execute(() -> method.invoke(serviceManager, args));
+        final RetryPolicy retryPolicy = new RetryPolicy().retryOn(NullPointerException.class)
```

Comment:
❗️ Since we've moved this retry logic in the tests instead of the code, we should replace failsafe with https://github.com/awaitility/awaitility. Sorry :-(